app-misc/qlcplus/qlcplus-4.14.1.ebuild

```diff
@@ -39,31 +36,28 @@ RDEPEND="
 IDEPEND="
 	dev-util/desktop-file-utils
 "
-DEPEND="${RDEPEND}
-	dev-qt/qttest:5
+DEPEND="
+	${RDEPEND}
 "
 
 src_prepare() {
 	default
 
-	sed -e "/UDEVRULESDIR/s:/etc/udev/rules.d:$(get_udevdir)/rules.d:" \
-		-i variables.pri || die
+	sed -e "s|lib/${CMAKE_C_LIBRARY_ARCHITECTURE}|$(get_libdir)|g" \
+		-i variables.cmake || die
 
-	## Remove Werror-flag since there are some warnings with gcc-9.x
-	sed -e "s/QMAKE_CXXFLAGS += -Werror/#&/g" \
-		-i variables.pri || die
+	sed -e "s|/etc/udev/rules.d|$(get_udevdir)|g" \
+		-i variables.cmake || die
 
-	## Remove Werror-flag since there are some warnings with gcc-9.x
-	sed -e "s/unix:QMAKE_CFLAGS += -Werror/#&/g" \
-		-i variables.pri || die
+	cmake_src_prepare
 }
 
 src_configure() {
-	eqmake5
-}
-
-src_install() {
-	emake INSTALL_ROOT="${D}" install
+	local mycmakeargs=(
+	)
+	# -DWITH_TESTS option works only with debug build, needs to be set here
+	# to not be overriden by cmake.eclass
+	CMAKE_BUILD_TYPE=Release cmake_src_configure
```

Comment:
I don't understand that section. Comment refers to "debug build, needs to be set here", but CMAKE_BUILD_TYPE is then set to Release (why that?) and neither does it do `-DWITH_TESTS`, so ...?

If we have to change that variable, we generally do it pre-inherit btw.